### ISSUE TITLE
Print a vue route with params

### ISSUE THREAD

**Issue by vargas11** [joseluisq/printd#17]
## Expected Behavior
 it's possible to printURL with params?

I have this dynamic endpoint `${process.env.API_URL}/invoice/${this.invoice.id}` but I got 404 not found, but if I change the url to a string said http://127.0.0.1:8080/invoice/13, I am able to see invoice.

    d.printURL(`${process.env.API_URL}/invoice/${this.invoice.id}`, ({ launchPrint }) => {
        console.log("Content loaded!");
        // fire printing!
        launchPrint();
      });
...

## Actual Behavior

...

## Steps to Reproduce the Problem

  1. ...
  1. ...
  1. ...

## Workaround

...

## Specifications

  - Version: `v0.0.0`
  - Platform: `v0.0.0`
  - Node version: `v0.0.0`
  - Browser version: `v0.0.0`

**Comment by joseluisq** [comment]
BTW Are you using maybe Chrome ? if so what version are you using?

**Comment by vargas11** [comment]
Yes I am using chrome Version 98.0.4758.80 (Official Build) (x86_64)

**Comment by joseluisq** [comment]
Looks like it's something with your `${process.env.API_URL}/invoice/${this.invoice.id}` endpoint and I see it as not related with Printd.
What I recommend is that you have to debug your endpoint and ensure it is valid or if you want you can share an example to be able to reproduce the issue.

**Comment by mech01nc01** [comment]
try:
```js
let url=`${process.env.API_URL}/invoice/${this.invoice.id}`;
console.warn('printing url: ', url);
d.printURL(url, ({ launchPrint }) => {
    console.log("Content loaded!");
    // fire printing!
    launchPrint();
  });
  ```
  what do you get from the console.warn?
